src/Firehose.Web/Authors/WilliamBarbosa.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class WilliamBarbosa : IAmAMicrosoftMVP
+    {
+        public string FirstName => "William";
+        public string LastName => "Barbosa";
+        public string StateOrRegion => "Santos, Brasil";
+        public string TwitterHandle => "willdotnet";
+        public string GitHubHandle => "willsb";
+
+        public string ShortBioOrTagLine
+            => "Mobile Developer, Microsoft MVP, Blogger, Speaker, Xamarin Enthusiast, Co-Founder/Host for the Monkey Nights Podcast/Webcast, MvvmCross contributor and usually the coolest hair in the room";
```

Comment:
Can you please shorten the bio, it breaks the layout.

Reply:
Done